sdk/template/template/src/universalEventEmitter.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// This node built-in must be shimmed for the browser.
+import EventEmitter from "events";
+// This is a node dependency that needs to be replaced with a
+// different implementation in the browser.
+import print from "./print";
+
+// this is a utility function from a library that should be external
+// for both node and web
+import { isNode } from "@azure/core-http";
+
+// exporting some value from a dependency
+export { URLBuilder } from "@azure/core-http";
+
+export function createEventEmitter(): EventEmitter {
+  // use event emitter
+  const e = new EventEmitter();
+
+  // Dynamic Node and browser-specific code
+  if (isNode) {
+    console.log("Node 👊");
+  } else {
+    console.log("Browser ❤");
```

Comment:
Thanks for preserving the emojis 
![image](https://sdk.bitmoji.com/render/panel/d9deca03-38d6-41d5-ac56-b1e902a13ac2-9afb776e-6872-4749-a379-f8d192e25a9c-v1.png?transparent=1&palette=1&width=150)